### PR TITLE
Fix docs errors and lint in ops

### PR DESCRIPTION
**Pull request by kevinsung**
- Fixed an error in the documentation of `QuadraticHamiltonian.bogoliubov_transform`.
- Removed lint caught by `pycodestyle` and my `pylint` in `ops`.

### CODE REVIEW

File: src/openfermion/ops/_givens_rotations_test.py [hunk 1]
```diff
@@ -171,7 +171,7 @@ def test_bad_dimensions(self):
         Q = Q[:m, :n]
 
         with self.assertRaises(ValueError):
-            givens_rotations, V, diagonal = givens_decomposition(Q)
+            _, _, _ = givens_decomposition(Q)
```
Comment (idk3) [inline src/openfermion/ops/_givens_rotations_test.py]: You don't need to unpack for these tests, `_ = ...` should be enough

File: src/openfermion/ops/_givens_rotations_test.py [hunk 2]
```diff
@@ -322,12 +322,10 @@ def test_bad_dimensions(self):
...
-                fermionic_gaussian_decomposition(rand_mat))
+            _, _, _ = fermionic_gaussian_decomposition(rand_mat)
 
     def test_bad_constraints(self):
         n = 3
         ones_mat = numpy.ones((n, 2 * n))
         with self.assertRaises(ValueError):
-            decomposition, left_unitary, antidiagonal = (
-                fermionic_gaussian_decomposition(ones_mat))
+            _, _, _ = fermionic_gaussian_decomposition(ones_mat)
```
Comment (idk3) [inline src/openfermion/ops/_givens_rotations_test.py]: `_ = ...` should be fine (also on L325)

File: src/openfermion/ops/_interaction_rdm_test.py [hunk 3]
```diff
@@ -13,9 +13,10 @@
 """Tests for interaction_rdms.py."""
 from __future__ import absolute_import
 
+import os
```
Comment (idk3) [inline src/openfermion/ops/_interaction_rdm_test.py]: `os` should be after `unittest` here I suspect

Comment (kevinsung) [reply src/openfermion/ops/_interaction_rdm_test.py]: They are both standard library modules so they should be placed in alphabetical order.

**Comment by kevinsung** [comment]
@idk3 I staled your review by removing unnecessary imports of `absolute_import`.

**Pull request merged**